circle.yml

```diff
@@ -1,3 +1,18 @@
 dependencies:
   pre:
     - sudo add-apt-repository -y ppa:ethereum/ethereum; sudo apt-get update; sudo apt-get install -y solc
+  override:
+    - echo "OK, let's go!"
+test:
+  override:
+    - sbt scalastyle
+    - sbt test:scalastyle
+    - sbt coverage test
+    - sbt coverage evm:test
+    - sbt coverageOff it:test
+    - sbt coverageReport coverageAggregate coveralls
+    - sbt dist
+  post:
+    - mkdir -p $CIRCLE_ARTIFACTS/scala-2.12
+    - mv target/scala-2.12/coverage-report  $CIRCLE_ARTIFACTS/scala-2.12/coverage-report
```

Comment:
I know this is not part of your task but as we have the chance....Are this files being used? Does coveralls need them? If not, maybe we should remove them, what do you think?